vexim/adminaliasaddsubmit.php

```diff
@@ -67,7 +67,7 @@
        ':localpart' => $_POST['localpart'],
        ':username' => $_POST['localpart'] . '@' . $_SESSION['domain'],
        ':domain_id' => $_SESSION['domain_id'],
-       ':crypt' => crypt_password($_POST['clear'],$salt),
+       ':crypt' => crypt_password($_POST['clear']),
```

Comment:
This (and siteaddsubmit.php) should have better been a separate PR. I wold have accepted it right away! :)
In future, please try making your PR's as atomic as possible – this makes the review and accepting process much faster, because good changes like this aren't then blocked by unrelated changes which require further discussion or editing.